packaging/chocolatey/crc/crc.nuspec.in

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>crc</id>
+    <version>__VERSION__</version>
+    <owners>https://github.com/crc-org</owners>
+    <title>CRC OpenShift 4</title>

```

Comment:
any suggestions on the title, it was initially `OpenShift Local` but then changed it to `CRC OpenShift 4`, should this be just `CRC` instead?

Reply:
`CRC` seems better than `CRC OpenShift 4`. If you want to refer to OpenShift, we should be using `Red Hat OpenShift Local`

Reply:
updated to `CRC - Runs Containers` as we are using that in the README also

Reply:
I thought we had a consensus that CRC Runs Containers wasn't that great, but this is creeping back lately :-(